README.md

```diff
@@ -1 +1,287 @@
-# GD_summit
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  <img src="img/logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>GD Summit </b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [GD_summit] <a name="about-project"></a>
+
+**[GD_summit]** is a project designed to inform the public about this year Graphic Designers Conference Scheduled to hold in Nigeria. 
+
+Here is a [presentation video](https://www.loom.com/share/6f51d68b44344618b18a5a8b1641429a)
```

Comment:
- [ ] @Donmark2k great work trying to implement the changes requested however I didn't see the section where you described the piece of code you find interesting or proud of making, you only showed the whole codebase, kindly ensure your presentation has a piece of code that you are proud of making or found interesting, also you suppose to add the video link in your pull request description
![image](https://user-images.githubusercontent.com/61408860/210961721-bc179a49-f0e1-4a79-b5e8-6ebb0925a238.png)

Reply:
Thank you @danmainah for the review.

Reply:
I have implemented this